DigitalLearningSolutions.Data/Models/TrackingSystem/PeriodOfActivity.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace DigitalLearningSolutions.Data.Models.TrackingSystem
+{
+    using System;
+    using DigitalLearningSolutions.Data.Enums;
+
+    public class DateInformation
+    {
+        public ReportInterval Interval { get; set; }
+        public DateTime? Date { get; set; }
+
+        public string GetDateLabel(bool shortForm)
+        {
+            string formatString;
+
+            var quarter = Date?.Month / 3 + 1;
+
+            switch (Interval)
+            {
+                case ReportInterval.Days:
+                    formatString = shortForm ? "y/M/d" : "yyyy/MM/d";

```

Comment:
Do we want all the date format strings here to be UK style if these end up being read somewhere? e.g. day/month/year?

Reply:
I want to say I got these from somewhere, but there's a D/M/Y example in the wireframes so I'll switch over.